sklearn/datasets/base.py

```diff
@@ -264,6 +270,8 @@ def load_iris():
         meaning of the features, and 'DESCR', the
         full description of the dataset.
 
+    (data,target) : tuple if return_X_y == True
```

Comment:
i prefer "...tuple if return_X_y is True", the `==` looks a bit odd to me. Additionally, i think it'd be nice to add to add two backticks (`) to the start and front of `return_X_y`.